airflow-core/src/airflow/decorators/__init__.py

```diff
@@ -16,15 +16,6 @@
 # under the License.
 from __future__ import annotations
 
-from airflow.sdk.definitions.decorators import (
-    TaskDecorator as TaskDecorator,
-    TaskDecoratorCollection as TaskDecoratorCollection,
```

Comment:
We don't import dag, task, taskgroup, teardown and such via that util, it will break back-compat

Reply:
Yep, accidentally removed them. Fixing

Reply:
Let's add `TaskDecoratorCollection` to `__deprecated_classes`

Reply:
Cool